src/bin/release-please.ts

```diff
@@ -41,46 +40,52 @@ interface YargsOptionsBuilder {
   option(opt: string, options: YargsOptions): YargsOptionsBuilder;
 }
 
+function releaseType(ya: YargsOptionsBuilder, defaultType?: string) {
+  const relTypeOptions: YargsOptions = {
+    describe: 'what type of repo is a release being created for?',
+    choices: getReleaserTypes(),
+  };
+  if (defaultType) {
+    relTypeOptions['default'] = defaultType;
```

Comment:
nit: `relTypeOptions.default`.

Reply:
ug, brain frazzle (when I get tired the python slips back in)